src/handlers/int-test-support/helpers/s3Helper.ts

```diff
@@ -54,11 +54,14 @@ const getS3Object = async (object: S3Object): Promise<string | undefined> => {
     Bucket: object.bucket,
     Key: object.key,
   };
-
-  const getObjectResult = await s3Client.send(
-    new GetObjectCommand(bucketParams)
-  );
-  return await getObjectResult.Body?.transformToString();
+  try {
+    const getObjectResult = await s3Client.send(
+      new GetObjectCommand(bucketParams)
+    );
+    return await getObjectResult.Body?.transformToString();
+  } catch (err) {
+    if (err instanceof Error) return err.name;
```

Comment:
what do you plan to use `err.name` for?

Reply:
@Si3rr4wow err.name is used in sad scenario 
<img width="665" alt="image" src="https://user-images.githubusercontent.com/112889496/215771587-3311549a-acfa-4fde-9a15-f4736adb8769.png">